selfdrive/car/docs_definitions.py

```diff
@@ -136,6 +143,7 @@ class CarInfo:
   min_steer_speed: Optional[float] = None
   min_enable_speed: Optional[float] = None
   harness: Enum = Harness.none
+  harness_kit_content: List[HarnessPart] = field(default_factory=lambda: [HarnessPart.harness_box, HarnessPart.comma_power_v2, HarnessPart.rj45_cable])
```

Comment:
Make this list a constant, btw the lambda will return the reference to the list, so any appends or removes will affect the list for all CarInfos. You'll want to return a copied list in the lambda